tegel/src/components/toast/toast.scss

```
@@ -10,8 +10,8 @@
   width: 88rem;
   min-height: var(--sdds-spacing-element-48);
 
-  &-icon svg {
-    margin: 4rem 0 0 3rem;
+  &-icon {
+    margin: 13.5px 0 0 3rem;
```

Please try to replace this one with:
```suggestion
    margin: 14px 0 0 12px;
```

I am not sure if 0.5px works on screens + let's have px values everywhere for consistency.